sidebar.php

```diff
@@ -26,7 +26,7 @@
 		<iframe src="//widgets.spinitron.com/widget/now-playing-v2?station=kbcs&num=5&sharing=1&cover=1&player=1&merch=0&non-music=1" width="100%" frameborder="0" allow="encrypted-media"></iframe>
 	</div>
 
-	<div id="social-links">
+	<nav aria-labelledby="social-links" id="social-links">
```

Comment:
An element can not be labelled by itself.

Reply:
Thanks for the catch, I have changed this to aria-label in a commit.